lib/experimental/class--wp-editors.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * Contains the placeholder class to replace the default `_WP_Editors`.
+ *
+ * @package gutenberg
+ * @since 6.3.0
+ */
+
+// phpcs:disable PEAR.NamingConventions.ValidClassName.StartWithCapital
+
+/**
+ * Placeholder class.
+ * Used to disable loading of TinyMCE assets.
+ *
+ * @access public
+ */
+final class _WP_Editors {
```

Comment:
Not sure I understand this part? How does loading this class prevents TinyMCE from loading?

Reply:
This, by default, is a core WP class that takes care of generating and "enqueueing" the editor assets. The problem is that for some of the scripts it doesn't use the traditional way to enqueue scripts, rather it will just output a huge blob of them inline, see:

https://github.com/WordPress/wordpress-develop/blob/559e6cecf49505ecc3a21fbff647fdc5c3f892a3/src/wp-includes/class-wp-editor.php#L1567

So the only way to not spit that blob of scripts out is to override this class. This is actually an optimization because none of what it does will even get enqueued that way (look around the class I linked above, it also enqueued some assets), so we won't need to dequeue them either. 

Luckily, redeclaring the class is possible because of the way that we check if the class is declared before being required, for example:

https://github.com/WordPress/wordpress-develop/blob/559e6cecf49505ecc3a21fbff647fdc5c3f892a3/src/wp-admin/includes/ajax-actions.php#L1923

and:

https://github.com/WordPress/wordpress-develop/blob/559e6cecf49505ecc3a21fbff647fdc5c3f892a3/src/wp-admin/includes/deprecated.php#L771

I realize this is a bold approach, but it makes it so straightforward to remove all editor assets that would otherwise be either enqueued or plainly spit out as inline scripts in the admin. This is one of the main reasons why I prefer having this whole thing wrapped as an experiment before releasing it to the public.